tutorials/RecSys22tutorial/03_Customize_Merlin_Models.ipynb

```diff
@@ -36,7 +36,7 @@
    "id": "ebc9c8c6-9fab-435f-958b-923b0f3d6ff4",
```

Comment:
<p>Should we add the image of DLRM to review? I think it is in the previous notebook, but it is might good to repeat it (not all details, just the diagram)</p><p> </p><p>Maybe provide a short overview, what are the steps, we will do?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/publications/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>ok.</p>